kythe/release/appengine/default/deploy.sh

```diff
@@ -18,8 +18,17 @@ MODULE=default
 
 cd "$(dirname "$0")"
 
+DEFAULT="$(gcloud app versions list --format=json --hide-no-traffic --service "$MODULE" 2>/dev/null | \
+  jq -r '.[].id')"
+
+echo "Current default version: $DEFAULT" >&2
+
 COMMIT="$(git rev-parse HEAD)"
-echo "Deploying Kythe website version $COMMIT" >&2
+if [[ "$DEFAULT" == "$COMMIT" ]];
+  echo "The website is already serving the latest version, skipping deployment..."
```

Comment:
It says that but I don't think the behavior changed?

Reply:
Good catch! Fixed it